tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -172,8 +173,20 @@ export class ScalarCardComponent<Downloader> {
           ...datum.metadata,
           closest: false,
           distSqToCursor: Math.hypot(
-            datum.point.x - cursorLoc.x,
-            datum.point.y - cursorLoc.y
+            datum.point.x - cursorLocationInDataCoord.x,
+            datum.point.y - cursorLocationInDataCoord.y
+          ),
+          distSqToCursorPixels: Math.hypot(
+            datum.point.x - cursorLocation.x,
+            datum.point.y - cursorLocation.y
+          ),
+          distSqToCursorX: Math.hypot(
```

Comment:
I think this is sqrt ((datum.point.x - cursorLocationInDataCoord.x)^2 + 0^2), which is just `datum.point.x - cursorLocationInDataCoord.x`. So this can be:

`distanceToCursorX: datum.point.x - cursorLocationInDataCoord.x`

Similar for distanceToCursorY.